File: src/CSharp/521.FindLUSlengthI.cs
using System;
using System.Linq;
using System.Collections.Generic;

namespace PeefyLeetCode.FindLUSlength
{
public class Solution {
    public int FindLUSlength(string a, string b) {
        return a == b ? -1 : Math.Max(a.Length, b.Length);
    }
}
}

File: src/Java/521.FindLUSlengthI.java

import java.awt.List;
import java.util.ArrayList;
import java.util.Arrays;
import java.util.Collections;
import java.util.Comparator;
import java.util.LinkedList;
import java.util.Queue;
import java.util.HashMap;

class Solution {
    public int findLUSlength(String a, String b) {
        return a == b ? -1 : Math.max(a.length(), b.length());
    }
}

